README.md

```diff
@@ -33,6 +33,8 @@ The following tools are currently available:
 - [`pds4_create_xml_index`](https://rms-pds4indextools.readthedocs.io/en/latest/pds4_create_xml_index.html):
   A command-line program to scrape information from a series of PDS4 XML labels, usually
   in a single collection, and generate a summary index file.
+- [`pds4_create_collection_index`](https://rms-pds4indextools.readthedocs.io/en/latest/pds4_create_collection_index.html):
+  A command-line program to create collection products for collections of PDS4 XML labels.
```

Comment:
Since this program only works on one collection at a time, let's say "to create a collection product from a collection of PDS4 XML labels".